plugins/pastefromword/filter/default.js

```diff
@@ -29,7 +29,8 @@
 			'v:group'
 		],
 		links = {},
-		inComment = 0;
+		inComment = 0,
+		convertToPxRegex = /\d+(\.\d+)?pt/g;
```

Comment:
Why this regex is here, as it could be inlined inside `convertToPx` function?

Reply:
It supposed to make some performance difference where regex object is created only once. However I tested it a bit and it looks like there is no difference or it's marginal, so it doesn't make sense to keep it separated so far from usage for readability.